packages/trader/src/Modules/Trading/Components/Form/TradeParams/strike.tsx

```diff
@@ -51,15 +53,24 @@ const Strike = observer(() => {
             <DesktopWrapper>
                 <Fieldset
                     className='trade-container__fieldset trade-container__barriers'
-                    header={localize('Strike price')}
+                    header={<Localize i18n_default_text='Strike price' />}
                     header_tooltip={
                         <Localize
                             i18n_default_text='If you buy a "<0>{{trade_type}}</0>" option, you receive a payout at expiry if the final price is {{payout_status}} the strike price. Otherwise, your “<0>{{trade_type}}</0>” option will expire worthless.'
                             components={[<strong key={0} />]}
                             values={{
-                                trade_type: contract_type === VANILLALONG.CALL ? localize('Call') : localize('Put'),
+                                trade_type:
+                                    contract_type === VANILLALONG.CALL ? (
+                                        <Localize i18n_default_text='Call' />
```

Comment:
Just asking: is `<Localize />` inside `<Localize />` working okay? But actually it's hart to check as Vanilla Call/Put is not translated to other languages anymore)

Reply:
It's not technically inside localize, it's a value inside and values do not get translated automatically but just inserted into parent translation, so yes, it needs to be translated separately.

Reply:
Actually, you're right, I remember why it didn't use component in the first place, it breaks the tooltip content

Reply:
I changed this part back to function to avoid any future issues